test/unit/logic/transaction.js

```diff
@@ -679,6 +679,17 @@ describe('transaction', function () {
 			});
 		});
 
+		it('should return error on timestamp out of int32 range', function (done) {
```

Comment:
below INT_32 range

Reply:
Also the second test checking the positive range exceed, 2147483648 + 1;

Reply:
> Also the second test checking the positive range exceed, 2147483648 + 1;

Already included :)